Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -1600,6 +1600,189 @@ final class LayerTests: XCTestCase {
     }
   }
 
+  func testBidirectionalBasicRNNSumMerge() {
+    // Data generated by "Tools/ReferenceImplementations/bidirectional_rnn.py".
+    // BEGIN GENERATED CODE
+    let kernelForward = Tensor<Float>(
+        [[-0.3603096,-0.5082418,0.06129855,0.106101096],
+        [-0.144324,0.5331736,-0.011734486,0.86278516],
+        [0.34075528,-0.64887196,0.36341316,0.28131205],
+        [0.12515205,-0.23425382,-0.1376664,0.22526532]]
+    )
+    let recurrentKernelForward = Tensor<Float>(
+        [[0.34720516,0.6280322,-0.6963875,-0.008289718],
+        [-0.057777617,-0.7034996,-0.6661205,0.240902],
+        [0.2612954,-0.2549268,-0.08859509,-0.9267621],
+        [-0.89879644,0.21372059,-0.25195017,-0.2881138]]
+    )
+    let biasForward = Tensor<Float>(
+        [0.0,0.0,0.0,0.0]
+    )
+    let kernelBackward = Tensor<Float>(
+        [[-0.5279052,0.4159922,-0.02112794,0.6500738],
+        [-0.16088444,-0.8408272,0.36293703,-0.23294312],
+        [0.14004618,0.69427365,-0.637702,0.077886045],
+        [0.3410867,-0.85888004,-0.23541403,0.1133005]]
+    )
+    let recurrentKernelBackward = Tensor<Float>(
+        [[0.67490876,-0.05608359,0.44851846,0.58325285],
+        [0.6672334,0.4354152,-0.21524309,-0.56469786],
+        [0.027053772,-0.4727077,0.6594652,-0.5838845],
+        [-0.31395176,0.7640783,0.5635665,0.003379479]]
+    )
+    let biasBackward = Tensor<Float>(
+        [0.0,0.0,0.0,0.0]
+    )
+    let x = Tensor<Float>(
+        [[[-0.8425845,-0.7054219,-0.023461938,0.35520858],
+          [-0.24800867,0.09574294,0.3316267,-0.13671225],
+          [-0.5058244,-0.6568735,-0.31837505,-0.7905574],
+          [-0.50511885,0.50999504,0.74102694,0.12891835]]]
+    )
+    let initialForwardLayerState = Tensor<Float>(
+        [[-0.22500783,0.3640524,-0.76411664,1.0557337]]
+    )
+    let initialBackwardLayerState = Tensor<Float>(
+        [[-0.9790854,-0.6808555,1.0346882,-1.0664041]]
+    )
+    let expectedSum = Tensor<Float>(
+      -1.3748394
+    )
+    let expectedStates = Tensor<Float>(
+        [[[0.039731443,0.30118462,-1.1362883,-0.6404417],
+          [-0.36877188,0.23013103,0.053439736,0.29040766],
+          [-0.7231241,0.4473409,-0.46913218,-0.64710104],
+          [0.41570857,-1.500107,-0.20221666,-0.08822429]]]
+    )
+    // TODO: Modify BidirectionalRecurrentLayer to return those two and test it
```

Comment:
The newly added tests have some unused value warnings:
```
tensorflow-swift-apis/Tests/TensorFlowTests/LayerTests.swift:1658:9: warning: initialization of immutable value 'expectedFinalStateForward' was never used; consider replacing with assignment to '_' or removing it
    let expectedFinalStateForward = Tensor<Float>(
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
    _
tensorflow-swift-apis/Tests/TensorFlowTests/LayerTests.swift:1661:9: warning: initialization of immutable value 'expectedFinalStateBackward' was never used; consider replacing with assignment to '_' or removing it
    let expectedFinalStateBackward = Tensor<Float>(
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
    _
tensorflow-swift-apis/Tests/TensorFlowTests/LayerTests.swift:2091:9: warning: initialization of immutable value 'expectedFinalStateHiddenForward' was never used; consider replacing with assignment to '_' or removing it
    let expectedFinalStateHiddenForward = Tensor<Float>(
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    _
tensorflow-swift-apis/Tests/TensorFlowTests/LayerTests.swift:2094:9: warning: initialization of immutable value 'expectedFinalStateCellForward' was never used; consider replacing with assignment to '_' or removing it
    let expectedFinalStateCellForward = Tensor<Float>(
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    _
tensorflow-swift-apis/Tests/TensorFlowTests/LayerTests.swift:2097:9: warning: initialization of immutable value 'expectedFinalStateHiddenBackward' was never used; consider replacing with assignment to '_' or removing it
    let expectedFinalStateHiddenBackward = Tensor<Float>(
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    _
tensorflow-swift-apis/Tests/TensorFlowTests/LayerTests.swift:2100:9: warning: initialization of immutable value 'expectedFinalStateCellBackward' was never used; consider replacing with assignment to '_' or removing it
    let expectedFinalStateCellBackward = Tensor<Float>(
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    _
tensorflow-swift-apis/Tests/TensorFlowTests/LayerTests.swift:2672:9: warning: initialization of immutable value 'expectedFinalStateForward' was never used; consider replacing with assignment to '_' or removing it
    let expectedFinalStateForward = Tensor<Float>(
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
    _
tensorflow-swift-apis/Tests/TensorFlowTests/LayerTests.swift:2675:9: warning: initialization of immutable value 'expectedFinalStateBackward' was never used; consider replacing with assignment to '_' or removing it
    let expectedFinalStateBackward = Tensor<Float>(
    ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
    _
```

Is it easy to test these values? If not, could you please comment them to silence the warnings?

Reply:
It is easy to return them but it seems kind of weird to me: I could modify `BidirectionalRecurrentLayer.callAsFunction` to return something like `(finalStateForward: Cell.TimeStepOutput, ..., output: Output)`. In Keras, it is done in a similar way if you check tests: https://github.com/tensorflow/swift-apis/pull/946/files#diff-5a1997b5e5dbf8509079fa2f4d5ae9d6R129. 

If you have an idea of how they should be returned I will implement it. For now, I commented out the variables.

Reply:
I see: commenting the tests sounds for now, thanks! I think the current versions of `callAsFunction` make sense for now.